Sources/Markdown/Parser/CommonMarkConverter.swift

```diff
@@ -299,7 +299,9 @@ struct MarkupParser {
         case CMARK_BULLET_LIST:
             return MarkupConversion(state: childConversion.state.next(), result: .unorderedList(parsedRange: parsedRange, childConversion.result))
         case CMARK_ORDERED_LIST:
-            return MarkupConversion(state: childConversion.state.next(), result: .orderedList(parsedRange: parsedRange, childConversion.result))
+            let cmarkStart = cmark_node_get_list_start(state.node)
+            let start: Int? = cmarkStart == 1 ? nil : Int(cmarkStart)
```

Comment:
Nit: Alternatively, we could make `start` non-optional and allow storing `1`. Then, clients would hardcode any specific behavior they want when start=1 (in DocC's case, to not emit the `start` property in render node).